.eslintrc

```diff
@@ -39,6 +39,7 @@
     // too much code.
     "@typescript-eslint/explicit-function-return-type": "off",
     "@typescript-eslint/no-explicit-any": "error",
+    "@typescript-eslint/explicit-module-boundary-types": "off",
```

Comment:
what does this rule do?

Reply:
It requires [explicit types for exported functions](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/explicit-module-boundary-types.md), which generates a ton of warnings. As mentioned in the comment above these rules, we rely on Typescript's ability to infer types, and if we allow this rule to be in place it would require us to add a lot of code to our files.  When also considering the fact that we are questioning the value of TS at all, I think that any rule that requires us to add a bunch of type code to our files should be avoided.